src/components/Records/Record/Organisations.vue

```diff
@@ -2,126 +2,148 @@
+          <v-card-text
+            v-for="(organisationLink, nodeIndex) in getRelations(key)"
+            :key="'organisationLink_' + nodeIndex"
+            :class="['pl-4 pl-2 pt-0 pb-0',{'pt-8':nodeIndex===0}]"
+          >
+            <v-card
+              class="pa-4 mt-2 d-flex flex-column v-card-hover"
+              flat
+              outlined
+            >
+              <a
+                :href="organisationLink.organisation.homepage"
+                target="_blank"
               >
-                <v-icon
-                  color="secondary"
-                  class="mr-2"
-                >
-                  fas fa-funnel-dollar
-                </v-icon>
-              </v-sheet>
-            </template>
-
-            <span>Grant funding this resource</span>
-          </v-tooltip>
-          <p class="ma-0">
-            {{ organisationLink.grant.name }}
-          </p>
-        </div>
-      </v-card>
+                {{ organisationLink.organisation.name }}
+              </a>
+              <p
+                v-if="organisationLink.organisation.types.length > 0"
+                class="ma-0"
+              >
+                ({{ organisationLink.organisation.types.join(', ') }})
+              </p>
+              <div
+                v-if="organisationLink.grant"
+                class="mt-4"
+              >
+                grants:
+                <strong>
+                  {{ organisationLink.grant.name }}
+                </strong>
+              </div>
+            </v-card>
+          </v-card-text>
+        </v-card>
+      </div>
     </div>
+    <section />
   </v-card>
 </template>
 
 <script>
-    import { mapGetters } from 'vuex';
-
-    import NoneFound from '@/components/Records/Record/NoneFound';
-    import SectionTitle from '@/components/Records/Record/SectionTitle';
-
-    /* TODO: Replace with query from database */
-    import organisationRelations from '@/data/organisationRelations.json';
-
-    export default {
-        name: "Organisations",
-        components: {
-            NoneFound,
-            SectionTitle
-        },
-        data(){
-          return {
-            /* TODO: Replace with query from database */
-            relations: organisationRelations
-          }
-        },
-        computed: {
-            ...mapGetters("record", ["getField"])
-        },
-        methods: {
-          getRelations(relName){
-            let _module = this;
-            let fields = _module.getField('organisationLinks');
-            return fields.filter(obj => obj.relation === relName);
-          }
-        }
+import { mapGetters } from 'vuex';
+import SectionTitle from '@/components/Records/Record/SectionTitle';
+/* TODO: Replace with query from database */
+import organisationRelations from '@/data/organisationRelations.json';
+export default {
+  name: "Organisations",
+  components: {
+    SectionTitle
+  },
+  data(){
+    return {
+      /* TODO: Replace with query from database */
```

Comment:
What's happening with this TODO, and the one at line 78?

Reply:
I don't know these were left from past. I did not touch those parts

Reply:
In which case they are probably left behind from old work, and would need to be removed.